src/test/java/com/coreos/jetcd/LeaseTest.java

```diff
@@ -20,45 +21,58 @@
   private Assertion test;
 
 
-  private ByteString testKey = ByteString.copyFromUtf8("foo1");
-  private ByteString testName = ByteString.copyFromUtf8("bar");
+  private static final ByteString KEY = ByteString.copyFromUtf8("foo");
+  private static final ByteString VALUE = ByteString.copyFromUtf8("bar");
 
   @BeforeTest
   public void setUp() throws Exception {
     test = new Assertion();
     client = ClientBuilder.newBuilder().endpoints(TestConstants.endpoints).build();
     kvClient = client.getKVClient();
-
     leaseClient = client.getLeaseClient();
   }
 
   @Test
   public void testGrant() throws Exception {
     long leaseID = leaseClient.grant(5).get().getID();
     PutResponse putRep = kvClient
-        .put(testKey, testName, PutOption.newBuilder().withLeaseId(leaseID).build()).get();
-    test.assertEquals(kvClient.get(testKey).get().getCount(), 1);
+        .put(KEY, VALUE, PutOption.newBuilder().withLeaseId(leaseID).build()).get();
+    test.assertEquals(kvClient.get(KEY).get().getCount(), 1);
 
     Thread.sleep(6000);
-    test.assertEquals(kvClient.get(testKey).get().getCount(), 0);
+    test.assertEquals(kvClient.get(KEY).get().getCount(), 0);
   }
 
   @Test(dependsOnMethods = "testGrant")
   public void testRevoke() throws Exception {
     long leaseID = leaseClient.grant(5).get().getID();
     PutResponse putRep = kvClient
-        .put(testKey, testName, PutOption.newBuilder().withLeaseId(leaseID).build()).get();
-    test.assertEquals(kvClient.get(testKey).get().getCount(), 1);
+        .put(KEY, VALUE, PutOption.newBuilder().withLeaseId(leaseID).build()).get();
+    test.assertEquals(kvClient.get(KEY).get().getCount(), 1);
     leaseClient.revoke(leaseID).get();
-    test.assertEquals(kvClient.get(testKey).get().getCount(), 0);
+    test.assertEquals(kvClient.get(KEY).get().getCount(), 0);
   }
 
   @Test(dependsOnMethods = "testRevoke")
+  public void testKeepAliveOnce() throws ExecutionException, InterruptedException {
+    long leaseID = leaseClient.grant(2).get().getID();
+    kvClient.put(KEY, VALUE, PutOption.newBuilder().withLeaseId(leaseID).build()).get();
+    test.assertEquals(kvClient.get(KEY).get().getCount(), 1);
+    Thread.sleep(1000);
+    leaseClient.keepAliveOnce(leaseID).get();
+    Thread.sleep(1500);
+    test.assertEquals(kvClient.get(KEY).get().getCount(), 1);
```

Comment:
remove 64-67. when we know server returns good response, we know keep alive request sent out correctly. we do not need to test server behavior in client test.